Makefile

```diff
@@ -339,7 +339,7 @@ test_collection_sanity:
 	if ! [ -x "$(shell command -v ansible-test)" ]; then pip install ansible-core; fi
 	ansible --version
 	COLLECTION_VERSION=1.0.0 make install_collection
-	cd $(COLLECTION_INSTALL) && ansible-test sanity --exclude=plugins/modules/export.py
+	cd $(COLLECTION_INSTALL) && ansible-test sanity --docker
```

Comment:
One request I'd like to make - I need a local debugging cycle, and it might be hard to do that with this. If you don't mind, I'd like to add a new Makefile arg, like `SANITY_OPTIONS` that will default to `--docker`, but in the CLI we can clear them out. Because this does a lot of stuff that's good to have automated for the feedback cycle, like installing and `cd`ing into the right directory, which is very annoying manually.

While, yes, sometimes I would like to run --docker locally, that takes a long time, and a parameter can also narrow the test subset.